### PR TITLE
Api improvements

### PR DESCRIPTION
**Pull request by wtrocki**
Changes tested in graphql-cli already. Mostly renaming to have unified patter for generators and also adding missing types

https://github.com/aerogear/graphback/issues/506

### CODE REVIEW

**Comment by wtrocki** [comment]
There is no way to verify changes as we simply making some minor type changes

File: packages/graphback-codegen-schema/src/createSchema.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+/**
+ * Create resolvers function that 
+ * @param inputContext 
+ * @param options 
+ */
+export const createSchema = (inputContext: InputModelTypeContext[], options: SchemaGeneratorOptions): string => {
+    if (options.format === 'ts') {
+        return new SchemaGenerator(inputContext, tsSchemaFormatter).generate();;
```
Comment (craicoverflow) [inline packages/graphback-codegen-schema/src/createSchema.ts]:
```suggestion
        return new SchemaGenerator(inputContext, tsSchemaFormatter).generate();
```

File: packages/graphback-codegen-schema/src/createSchema.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }
+    if (options.format === 'js') {
+        return new SchemaGenerator(inputContext, jsSchemaFormatter).generate();;
```
Comment (craicoverflow) [inline packages/graphback-codegen-schema/src/createSchema.ts]:
```suggestion
        return new SchemaGenerator(inputContext, jsSchemaFormatter).generate();
```

File: packages/graphback-codegen-schema/src/createSchema.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    }
+    if (options.format === 'gql') {
+        return new SchemaGenerator(inputContext, gqlSchemaFormatter).generate();;
```
Comment (craicoverflow) [inline packages/graphback-codegen-schema/src/createSchema.ts]:
```suggestion
        return new SchemaGenerator(inputContext, gqlSchemaFormatter).generate();
```

File: packages/graphback-codegen-schema/src/createSchema.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    }
+    throw Error("Invalid format specified. `options.format` supports only `ts` and `js` flags ");
```
Comment (craicoverflow) [inline packages/graphback-codegen-schema/src/createSchema.ts]:
```suggestion
    throw Error("Invalid format specified. `options.format` supports only `ts` `js` and `gql` flags ");
```

**Pull request merged**